repobee_feedback/feedback.py

```diff
@@ -11,13 +11,15 @@
 import re
 import sys
 import argparse
+from textwrap import indent
 from typing import Iterable, Tuple, List, Mapping
 
 import repobee_plug as plug
 
 PLUGIN_NAME = "feedback"
 
 BEGIN_ISSUE_PATTERN = r"#ISSUE#(.*?)#(.*)"
+INDENTATION_STR = "    >"
```

Comment:
After looking at it, I don't think I'm too crazy about the `>`, especially as it has meaning in Markdown (which matters here, as issue bodies are formatted with Markdown on all supported platforms). How about we just go with pure indentation (spaces or a single tab)?

Reply:
Sure!